Makefile

```diff
@@ -872,6 +872,20 @@ $(if $(PRECISION),$(info NOTE: PRECISION=single is unnecessary, single is the de
 	PRECISION_MESSAGE="MPAS was built with default single-precision reals."
 endif #PRECISION IF
 
+# Optional MUSICA support for chemistry
+ifeq "$(shell echo $(MUSICA) | tr '[:upper:]' '[:lower:]')" "true"
+ifeq ($(shell pkg-config --exists musica-fortran && echo yes || echo no), no)
+$(error "musica-fortran package is not installed. Please install it to proceed.")
+endif
+	MUSICA_FCINCLUDES += $(shell pkg-config --cflags musica-fortran)
+	MUSICA_LIBS += $(shell pkg-config --libs musica-fortran)
+	MUSICA_FFLAGS = -DMPAS_USE_MUSICA
+endif
+
+FCINCLUDES += $(MUSICA_FCINCLUDES)
+LIBS += $(MUSICA_LIBS)
+FFLAGS += $(MUSICA_FFLAGS)
```

Comment:
Could these three lines be moved inside the `ifeq` block above, since the MUSICA variables are only set if `MUSICA=true` was specified?

Reply:
moved!